moveit_ros/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -99,7 +99,7 @@ void OccupancyMapMonitor::initialize()
                                                      << "\" specified but no TF instance (buffer) specified. "
                                                         "No transforms will be applied to received data.");
 
-  tree_.reset(new OccMapTree(map_resolution_));
+  tree_.reset(new collision_detection::OccMapTree(map_resolution_));
```

Comment:
You need to change the namespace in line 44 instead.

Reply:
no? the wrapper `OccMapTree` was moved out of this namespace/module but occupancy_map_monitor is still there